src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -159,15 +180,23 @@ export default function TokenTable({
     }
   }, [maxItems, tokensData])
 
+  useEffect(() => {
```

Comment:
what is this for? does it auto scroll the table row? maybe add a comment hwat it does

Reply:
Added comment